app/lint-baseline.xml

```diff
@@ -1,6 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <issues format="5" by="lint 3.5.0" client="gradle" variant="all" version="3.5.0">
 
+    <issue
```

Comment:
This is a false positive. The style is used by the community login activity layout.

Reply:
See my comment below. I do not think this is a false positive. The issue states that the resource is unused in some variants which is most certainly true.